src/utils.js

```diff
@@ -32,6 +32,17 @@ export function singleLineString(strings, ...vars) {
   }).join(' ').trim();
 }
 
+export function getVariable(context, name) {
+  var variables = context.getScope().variables;
+  var result;
+  variables.forEach(function(variable) {
```

Comment:
Comments here to explain the eslint apis being used and what it's doing are particularly important here.